project-base/storefront/components/Blocks/SimpleNavigation/SimpleNavigationListItem.tsx

```diff
@@ -21,11 +21,28 @@ export const SimpleNavigationListItem: FC<SimpleNavigationListItemProps> = ({
     const itemImage = 'mainImage' in listedItem ? listedItem.mainImage : null;
     const href = getStringWithoutTrailingSlash(listedItem.slug) + '/';
 
+    const getLinkType = (type: ListedItemPropTypeTypename | undefined): PageType => {
+        switch (type) {
+            case 'ArticleSite':
+                return 'article';
+            case 'BlogArticle':
+                return 'blogArticle';
+            case 'Brand':
+                return 'brand';
+            case 'Category':
+                return 'category';
+            default:
+                throw new Error('Link type ' + type + ' could not be resolved.');
+        }
+    };
+
+    const resolvedLinkType = linkType ?? getLinkType(listedItem.__typename);
```

Comment:
Let's have this as simple `linkType` variable and rename `linkType` from props as `defaultLinkType`. You can do so like

`export const SimpleNavigationListItem: FC<SimpleNavigationListItemProps> = ({
    listedItem,
    linkType: defaultLinkType,
    tid,
    className,
}) => {`